pq/README.md

```diff
@@ -91,3 +98,21 @@ The client will be prompted for a message to send to the server. Once you see
 this prompt, a quantum-safe connection has already been established. Use the
 client to send the message "shutdown" in order to end the execution of the
 server.
+
+
+## Signing and Verifying a Message with LMS/HSS
+
+Run the `lms_sign_verify` example without options to see its help and usage:
```

Comment:
The commit message has a lot of great content that should be moved here.

Reply:
I moved the LMS/HSS example into a subdirectory (`pq/stateful_hash_sig`), and added more to its readme.

This also made it easier to separate out building with hash-sigs vs liboqs in the Makefile.